modules/service/src/main/resources/graphql/itc.graphql

```diff
@@ -14939,19 +14951,22 @@ union TargetIntegrationTimeOutcome = TargetIntegrationTime | TargetError
 # ITC results for a target, can be a single result or one per CCD
 type TargetIntegrationTime {
   # All CCD results for the target
-  all: [IntegrationTime!]!
+  all: [IntegrationTime!]
 
   # Index of the preferred CCD result, it is mode dependent
-  index: NonNegInt!
+  index: NonNegInt
 
   # Preferred CCD result for the target
-  selected: IntegrationTime!
+  selected: IntegrationTime
 
   # The band used for the time calculation, in case of a BandNormalized SED
   band: Band
 
   # The emission line used for the time calculation, in case of an EmissionLines SED
   emissionLine: Wavelength
+
+  # calculated signal to noise at a specific wavelength. Returned only if the wavelength At is valid
+  signalToNoiseAt: SignalToNoiseAt
```

Comment:
This is now a property of the target and not at the CCD level. The value can be null if you rrequst a wavelength outside of the range of the detectors